MeetU/MeetU/Scripts/Controllers/Profile/profile-img-edit.js

```diff
@@ -0,0 +1,48 @@
+﻿(function () {
+    "use strict";
+    angular
+        .module('meetupModule')
+        .controller('ProfileImageEditController', ProfileImageEditController);
+    ProfileImageEditController.$inject = ["$scope", "$resource", "$location"];
+    function ProfileImageEditController($scope, $resource, $location) {
+        var vm = this;
+        var User = $resource('/api/loggedUser');
+        var ProFilePicture = $resource('api/ProfilePicture');
+
+        User.query(function (user) {
+            vm.loggedUser = user[0];
+        });
+
+        $scope.Image = '';
+        $scope.CroppedImage = '';
+        var handleFileSelect = function (event) {
+            var file = event.currentTarget.files[0];
+            var reader = new FileReader();
+            reader.onload = function (event) {
+                $scope.$apply(function ($scope) {
+                    $scope.Image = event.target.result;
+                });
+            };
+            reader.readAsDataURL(file);
+        };
+        angular.element(document.querySelector('#fileInput')).on('change', handleFileSelect);
```

Comment:
should change #fileinput to a class with prefix. we should avoid a common used name like this.